.github/workflows/chatbot.yaml

```diff
@@ -8,6 +8,8 @@ on:
       - ./recipes/common/Makefile.common
       - ./recipes/natural_language_processing/chatbot/**
       - .github/workflows/chatbot.yaml
+      - ./recipes/natural_language_processing/chatbot/app/**
```

Comment:
Isnt this covered by [line 9](https://github.com/containers/ai-lab-recipes/pull/548/files#diff-fbe2aff97f825e5dbb33a5177bf360bb520df8fdecb75dcd3f025fb02a737e24R9)?

Reply:
@Gregory-Pereira I was hoping it would be but because the tests didn't run when we poked the requirements file I worry we may need the redundancy

Reply:
Thats fine, im more concerned with functionality than redundancy. If there is even a question about it feel free to leave it